cartography/util.py

```diff
@@ -37,6 +37,18 @@
 STATUS_KEYBOARD_INTERRUPT = 130
 DEFAULT_BATCH_SIZE = 1000
 
+def get_azure_resource_group_name(id:str)->None:
+    resource_group=''
+    if id is not None:
+        x = id.split('/')
+        if 'resourcegroups' in x:
```

Comment:
instead of 2 if conditions. convert x to lowercase and compare with lower case `resourcegroups`